sh/gtf/validate_genome_references.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+from ftplib import FTP
+import argparse, re, os, sys
+
+fail = False
+def parse_url(url):
+    """
+    Parses a complete ftp URL into server, path and file name.
+    """
+    match = re.search("ftp://([a-z\.]*)/(.*)$", url)
+    server = match.group(1)
+    path_tokens = match.group(2).split("/")
+    return server, "/"+"/".join(path_tokens[:-1]), path_tokens[-1]
+
+def check_connection(organism, server, corrected_url, path, file):
+    ftp = FTP(server)
+    ftp.login()
+    
+    try:
+        ftp.cwd(path)
+    except Exception as e:
+        if "550 Failed to change directory" in str(e):
+            parent_path = '/'.join(path.rstrip('/').split('/')[:-2])
+
+            if file.endswith((".gtf", ".gtf.gz")):
```

Comment:
How would you consider .GTF and .GTF.GZ?
```suggestion
            GTF_EXTENSIONS = (".gtf", ".gtf.gz")
            if file.lower().endswith(GTF_EXTENSIONS):
```